### PR TITLE
denemeceler denemeceler

### PR DESCRIPTION
**Pull request by Ece-Kahraman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ h.
 **Design Report:** 99\
```
Comment (Ece-Kahraman) [inline README.md:21]: wha

File: codes/add_new_card.php [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?php
+//ehuehue
```
Comment (Ece-Kahraman) [inline codes/add_new_card.php:2]: ACTIVE ISSUE (NON-BLOCKING): this is non blocking

**Review by Ece-Kahraman — COMMENTED** [review]
hello

File: README.md [hunk 3]
```diff
@@ -14,7 +14,6 @@ system reports like finding the most popular events in a month based on the numb
 people who attended etc.
```
Comment (Ece-Kahraman) [inline README.md:15]: yoo

**Comment by Ece-Kahraman** [comment]
**Closed**: updated again

**Comment by Ece-Kahraman** [comment]
**Outdated**: /github.com/Ece-Kahraman/Eventium/pull/5#issuecomment-2027194192 

hello

**Comment by Ece-Kahraman** [comment]
> 
> bupdated again
> https://github.com/Ece-Kahraman/Eventium/pull/5#issuecomment-2027194192 

hello
> https://github.com/Ece-Kahraman/Eventium/pull/5#issuecomment-2041472834 

yoo

**Comment by Ece-Kahraman** [comment]
yo

**Comment by Ece-Kahraman** [comment]
**ACTIVE**: <p>denedim</p>

**Comment by vIremAydin** [comment]
**ACTIVE**: <ol><li><p><sub><mark>dd</mark></sub></p></li></ol>

**Comment by vIremAydin** [comment]
**ACTIVE**: <p style="text-align: right">eded</p>

**Comment by vIremAydin** [comment]
**ACTIVE**: <h3>dedededded</h3>

**Pull request closed without merging**